portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,13 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/** Fetches the current date from the server and adds it to the page. */
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+async function showMessage() {
+  const responseFromServer = await fetch('/hello');
+  const textFromResponse = await responseFromServer.json();
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const messageContainer = document.getElementById('message-container');
+  var randomNum = Math.floor(Math.random() * 4);
```

Comment:
Instead of hardcoding the number 4, you can do textFromResponse.length so it will work for any amount

Reply:
Sounds good!